1.1.11

PBruk opened the pull request:
Changes to card structure

matt-bernhardt commented:
This change looks like it is mostly focused on stripping out a lot of the conditional programming around excerpts, with a bit of CSS adjustments?

PBruk commented:
Correct. I removed lots of conditionals. Also, I reset the height - and width, via HTML - of boxes across all media sizes; restyled the font-sizes; added space for subtitles; reconfigured the padding in between card elements; added a wrapper within the cards to ensure there is no overflow; etc.

matt-bernhardt commented:
This has been deployed to test.
